automation/src/test/scala/org/broadinstitute/dsde/test/api/AuthDomainGroupApiSpec.scala

```diff
@@ -150,6 +154,31 @@ class AuthDomainGroupApiSpec extends AnyFreeSpec with Matchers with WorkspaceFix
       }
 
     }
+
+    "bucket should not be accessible to project owners via projectViewer Google role" in {
+
+      val userA = UserPool.chooseProjectOwner //The project owner who can't see the workspace
+      val userB = UserPool.chooseAuthDomainUser //The user who owns the workspace
+
+      val userAToken: AuthToken = userA.makeAuthToken()
+      val userBToken: AuthToken = userB.makeAuthToken()
+
+      withGroup("AuthDomain", List(userB.email)) { authDomainName =>
+        withCleanBillingProject(userA, List(userA.email), List(userB.email)) { projectName =>
+          withWorkspace(projectName, "AuthDomainGroupApiSpec_workspace", Set(authDomainName)) { workspaceName =>
+
+            val bucketName = Rawls.workspaces.getWorkspaceDetails(projectName, workspaceName)(userBToken).parseJson.convertTo[WorkspaceResponse].workspace.bucketName
+
```

Comment:
for safety, add an assert here that `userB` _*can*_ access the bucket, and gets a 200 from `getBucket`. That will verify that the bucket was created in good shape and accessible to the owner. This should prevent any false-positives wherein something went wrong with the bucket and `userA` gets a 403 for the wrong reason.

Reply:
haha see my comment below